lib/metrics.js

```diff
@@ -1,5 +1,5 @@
 var Statsd = require('strong-statsd');
-var agent = require('./agent');
+// var agent = require('./agent');
```

Comment:
metrics is called from config, so there is no way to avoid this change.
config.js initializes sendMetrics with an instance of this object.